cvat-ui/src/reducers/server-api-reducer.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { BoundariesActions, BoundariesActionTypes } from 'actions/boundaries-actions';
+import { ServerAPIActions, ServerAPIActionTypes } from 'actions/server-actions';
+import { ServerAPIState } from '.';
+
+const defaultState: ServerAPIState = {
+    schema: null,
+    fetching: false,
+    initialized: false,
+    configuration: {
+        isRegistrationEnabled: false,
+        isBasicLoginEnabled: false,
```

Comment:
I belive those two shouldnt be disabled by default. It breaks login form if something goes wrong on the server and schema request fails:
![image](https://github.com/opencv/cvat/assets/50956430/1c6e4355-0a0e-4309-a437-82b95e5e888b)
It was only about change/reset

Reply:
@klakhov, I disagree with you because I guess we should use the same default values for the endpoint's configuration. It does not matter if we cannot identify enabled login/register endpoints or reset/change password endpoints. We have enabled by default these endpoints on the server (on the public version). If we enable all endpoints by default on the client it will improve user experience if the schema request fails but it can render UI not based on the server configuration. Looks like the first option is more preferable

Reply:
Ok, I agree. Disabling form worsens UX. Anyway, the request will fail if the setting is disabled.